docs/source/future/core/nrunner.rst

```diff
@@ -46,6 +56,235 @@ implemented under a different architecture and implementation:
 * Simplified and automated deployment of the runner component into
   execution environments such as containers and virtual machines
 
+Components of the Current and N(ext) Runner architecture
```

Comment:
> As far as I understood, this head refers to all the content until the end of this PR. In this case, this head should be the main section, and the others, listed below, should be subsections.

I just looked at the text in a formatted way and not the "code" here and I changed my mind. I agree with @beraldoleal that the head should be slightly changed. My suggestion would be "Current and N(ext) Runner components of Avocado".

Reply:
This head is a second level head and is alone.

Reply:
OK, I'm a bit confused here... I can change this to whatever you guys think make more sense, because the goal is really to make sense to readers (the writer doesn't matter much here).  We have:

* Avocado's current state of the art - @beraldoleal 
* Current and N(ext) Runner components of Avocado - @willianrampazzo 

Let me know if we have a consensus on any of them.

Reply:
I'm ok with both.

Reply:
When I read 'state of the art', I think this is the latest and greatest, but with more and more updates coming, the documentation will start to get obsolete and will not reflect the 'state of the art' anymore. @beraldoleal what do you think?

Reply:
Because we may need a tie breaker, and @beraldoleal is OK with @willianrampazzo proposal, I'm picking "Current and N(ext) Runner components of Avocado". Thanks!